sycl/source/queue.cpp

```diff
@@ -25,6 +25,7 @@ __SYCL_INLINE_VER_NAMESPACE(_V1) {
 queue::queue(const context &SyclContext, const device_selector &DeviceSelector,
              const async_handler &AsyncHandler, const property_list &PropList,
              Discriminator Disc) {
```

Comment:
could do `[[maybe_unused]] Discriminator Disc`

Reply:
I don't see this parameter in the spec - https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#_queue_interface. So we added a parameter, which we ignore. Could you please clarify why do we ignore `Disc`?

Reply:
See comment in #7196.